yellowbrick/gridsearch/pcolor.py

```diff
@@ -0,0 +1,225 @@
+"""
+Colorplot visualizer for gridsearch results.
+"""
+
+import time
+import numpy as np
+
+from .base import GridSearchVisualizer
+from ..exceptions import YellowbrickValueError
+
+from sklearn.metrics.pairwise import pairwise_distances
+from sklearn.preprocessing import LabelEncoder
+from matplotlib.mlab import griddata
+
+
+## Packages for export
+__all__ = [
+    "GridSearchColorPlot",
+    "gridsearch_color_plot"
+]
+
+
+##########################################################################
+## Dimension reduction utility
+##########################################################################
+
+def param_projection(cv_results, param_1, param_2):
```

Comment:
This function seems extremely useful for all `GridSearchVisualizers` - perhaps this should be in `base.py` -- possibly as a method of `GridSearchVisualizer`?

Reply:
Or as a standalone function, with an aliased (static?) method in `GridSearchVisualizer`?

Reply:
Also: future work - add more params?

Reply:
Alright, I moved it up to the base module and aliased it in the base GridSearchVisualizer class.  I didn't make the aliased method static, rather letting it access the gridsearch model object as `self.estimator`.

Definitely could add more params as future work.